HW15/src/FileLoggerConfigurationLoader.java

```diff
@@ -0,0 +1,35 @@
+import java.io.FileNotFoundException;
+import java.text.ParseException;
+import java.util.Date;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Scanner;
+
+public class FileLoggerConfigurationLoader {
```

Comment:
FileLoggerConfigurationLoader was implemented incorrectly - the goal of this class is to read FileLoggerConfiguration from the specified file and return the FileLoggerConfiguration object.